aspnetcore/security/docker-https.md

```diff
@@ -36,7 +36,7 @@ For production certs:
 * The `dotnet dev-certs` tool is not required.
 * Certificates do not need to be stored in the location used in the instructions. Any location should work, although storing certs within your site directory is not recommended.
 
-The instructions volume mount certificates into containers. You can add certificates into container images with a `COPY` command in a *Dockerfile*. Copying certificates into an image isn't recommended for the following reasons:
+The instructions volume mount certificates into containers. You could add certificates into container images with a `COPY` command in a *Dockerfile*, but it's not recommended. Copying certificates into an image isn't recommended for the following reasons:
```

Comment:
Kia ora, @Rick-Anderson! I love the work you've done here, definately an improvement 👍 .  I'm a little confused still by the sentence , "The instructions volume mount certificates into containers." Is that meant to refer to instructions within *this current document*?

Can I suggest:

"The instructions contained below in this document volume mount certificates into containers, using Docker's `-v` command-line option."

Reply:
That looks good. I'll close this and you can PR and fix, that way you'll get the credit. 

 At the top right of the article, select the Edit pen icon:

![image](https://user-images.githubusercontent.com/3605364/53368931-11972200-38ee-11e9-86ce-18aaff69c719.png)

Select the pen icon again. Save.

Reply:
Too kind! But, thank you!